ecosystem/python/sdk/aptos_sdk/account_address.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+import hashlib
+import io
+import typing
+import unittest
+
+from bcs import Deserializer, Serializer
+import ed25519
+
+
+class AccountAddress:
+    address: bytes
+    LENGTH: int = 32
+
+    def __init__(self, address: bytes):
+        self.address = address
+
+        if len(address) != AccountAddress.LENGTH:
+            raise Exception("Expected address of length 32")
+
+    def __eq__(self, other: AccountAddress) -> bool:
+        return self.address == other.address
+
+    def __str__(self):
+        return self.hex()
+
+    def hex(self) -> str:
+        return self.address.hex()
+
+    def from_hex(address: str) -> AccountAddress:
+        addr = address
```

Comment:
I might add a comment saying what this takes in, both `0x` prefixed, and shortened hex.

Reply:
comments be damned for now, save that for future work, so it doesn't come off as half assed...